vendor/renanbr/bibtex-parser/tests/resources/valid/multiples-entries.bib

```diff
@@ -0,0 +1,5 @@
+% comment #1 must be ignore
+@entryFooWithSpaces { foo = oof } % comment #2 must be ignore
+% comment #3 must be ignore
+@entryBarWithoutSpaces{bar=rab}% comment #4 must be ignore
```

Comment:
made an edit, then undid it as I am not sure if the layout for line 4 is to be the same as the layout on line 2.

Reply:
The vendor folder is auto generated, I didn't write any of that code. If it's okay I will merge it now.